lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -572,8 +574,21 @@ def make_node_file(self):
             self.log.raiseException(msg)
 
     def get_universe_ncpus(self):
-        """Return ppn for universe"""
-        return self.mpiprocesspernode
+        """Return ppn dict for universe"""
```

Comment:
@Caylo better: `Construct dictionary with number of processes to start per node, based on --universe`